app/build.gradle

```
@@ -80,11 +80,17 @@ dependencies {
     implementation 'io.github.inflationx:calligraphy3:3.1.1'
     implementation 'io.github.inflationx:viewpump:2.0.3'
 
+    // preference
+    implementation("androidx.preference:preference-ktx:1.1.0")
+
+    // coroutines
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
+
     //anko
-    implementation 'org.jetbrains.anko:anko-sdk25:0.10.6'
     implementation 'org.jetbrains.anko:anko-sqlite:0.10.6'
-    implementation 'org.jetbrains.anko:anko-support-v4:0.10.6'
-    implementation 'org.jetbrains.anko:anko-appcompat-v7:0.10.6'
+
+    // Splitties
+    implementation("com.louiscad.splitties:splitties-fun-pack-android-appcompat:3.0.0")
```

Not a big problem, but why replace Anko by another library here ? I think it's always better to stick to the framework as much as possible to avoid removing a library because it's not maintained anymore... just like Anko.

---

Yes you're right, but the alertdialog from splitties (similar to the one from Anko) is very convenient and easy to read compared to the default alertdialog builder.

---

It's a bit overkill seeing the extent of the library if it's just for the alertdialog syntaxe ^^'
Just adding an extension function would do the job.

---

```suggestion
    implementation("com.louiscad.splitties:splitties-alertdialog-appcompat:3.0.0")
```
This should only include the alertdialog

---

Nice !